modules/op-improving-performance.adoc

```diff
@@ -0,0 +1,20 @@
+// This module is included in the following assemblies:
+// * resource/managing-pipelines-performance.adoc
+
+:_content-type: PROCEDURE
+[id="improving-performance_{context}"]
+= Improving {pipelines-shortname} performance
+
+If you experience slowness or recurrent failures of pipeline runs, you can take any of the following steps to improve the performance of {pipelines-shortname}.
+
+* Monitor the resource usage of the nodes in the {OCP} cluster on which {pipelines-shortname} runs. If the resource usage is high, increase the number of nodes.
++
+* Enable high-availability mode. This mode affects the controller that creates and starts pods for task runs and pipeline runs. In Red Hat testing, high-availability mode significantly reduced pipeline execution times as well as the delay from creating a `TaskRun` resource CR to the start of the pod executing the task run. To enable high-availability mode, make the following changes in the `TektonConfig` custom resource (CR):
```

Comment:
I'm not very familiar with Pipelines, but this "procedure" seems to assume a lot of knowledge. While I am okay with approving this for now, I would like to see improvements in the future, showing an example CR and specs with callouts to where these fields are and how to adjust them. 

If you have time, this would be a great improvement to make today. Otherwise, I would prioritize improving this in the near term.